.github/DISCUSSION_TEMPLATE/player-support.yml

```diff
@@ -0,0 +1,61 @@
+labels:
+  - help wanted
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        ## **Where are my Logs?**
+
+
+
+        If your game is crashing, you should definitely upload your logs!
+
+
+
+         Check out the following pages for more information on how to find your
+        log files.
+        
+        
+        
+        - [Finding Your Logs on Windows - Fabric
+        Wiki](https://fabricmc.net/wiki/player:tutorials:logs_ml:windows)
+
+        - [Finding Your Logs on MacOS - Fabric
+        Wiki](https://fabricmc.net/wiki/player:tutorials:logs_ml:mac)
+        
+        
+
+        It is recommended you upload your log files to https://mclo.gs/ to allow
+        easily readability and to censor any sensitive information such as
+        client tokens or personal information exposed by your operating system.
```

Comment:
I have never used this website thus dont really know how much I trust it. I dont think we should be guarenteing that it removes personal infomation.

The logs should never contain personal info, maybe other than your computer's username?

Reply:
Gist might be an alternative, they are already using github anyway.